erpnext/edi/doctype/common_code/common_code.py

```
@@ -28,6 +28,10 @@ class CommonCode(Document):
 		title: DF.Data
 	# end: auto-generated types
 
+	def before_validate(self):
+		if self.title and len(self.title) > 140:
+			self.title = self.title[:139] + "…"
+
```

If the title is used as canonical reference value (which I know I do), this might just start **randomly** braking some XML validations.

I'd construe this to be a view, not a data issue.

---

What's the alternative?

---

We'd have essentially two alternatives:

- Let it fail an the current field length (eventually with a better error message)
- Let it fail (very much less likely so) on an increased field length (eventually with a better error message), e.g. of `length: 300`

---

We could instruct the user tho dial up the field length on failure via customization, for example.

---

Increased the length, will add the error message as is.